src/__test__/components/data-exploration/CellInfo.test.jsx

```diff
@@ -1,25 +1,18 @@
 import React from 'react';
-import { Provider } from 'react-redux';
 import { mount } from 'enzyme';
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
 import { Card } from 'antd';
 import CellInfo from '../../../components/data-exploration/CellInfo';
 import '__test__/test-utils/setupTests';
 
-const mockStore = configureMockStore([thunk]);
+const cellInfo = {
+  cellId: 1904,
+  componentType: 'heatmap',
+  expression: 0,
+  geneName: 'DOK3',
+};
 
 describe('CellInfo', () => {
   test('renders correctly when hovering over the same component', () => {
-    const store = mockStore({
-      cellInfo: {
-        cellName: 1904,
-        componentType: 'heatmap',
-        expression: 0,
-        geneName: 'DOK3',
-      },
-    });
-
     const coordinates = {
```

Comment:
Can you take out these `coordinates` and put them out of the test blocks so it can be defined once and used by all the tests?